### PR TITLE
tageditor: Add version 3.4.2

### PR DESCRIPTION
**Pull request by KaranKad**
Added new application

### CODE REVIEW

File: bucket/tageditor.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "license": "GPL-2.0-only",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/Martchus/tageditor/releases/download/v3.4.2/tageditor-3.4.2-x86_64-w64-mingw32.exe.zip",
+            "hash": "ff4725dd63f5b8961dedd051c4b22f635d791f337adc2fc43c576e93c9250a5c"
+        }
+    },
+    "shortcuts": [
+        [
+            "tageditor-3.4.2-x86_64-w64-mingw32.exe",
```
Comment (Ash258) [inline bucket/tageditor.json:14]: Having versioned executable in shortcuts in manifest, which has autoupdate is really not good idea...

File: bucket/tageditor.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+    "version": "3.4.2",
+    "description": "A tag editor with Qt GUI and command-line interface supporting MP4/M4A/AAC (iTunes), ID3, Vorbis, Opus, FLAC and Matroska.",
+    "homepage": "https://github.com/Martchus/tageditor",
+    "license": "GPL-2.0-only",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/Martchus/tageditor/releases/download/v3.4.2/tageditor-3.4.2-x86_64-w64-mingw32.exe.zip",
+            "hash": "ff4725dd63f5b8961dedd051c4b22f635d791f337adc2fc43c576e93c9250a5c"
+        }
```
Comment (Ash258) [inline bucket/tageditor.json:10]: Where is 32bit veersion??

File: bucket/tageditor.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    },
+    "shortcuts": [
```
Comment (Ash258) [inline bucket/tageditor.json:12]: https://github.com/Martchus/tageditor#cli

File: bucket/tageditor.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+        [
+            "tageditor-3.4.2-x86_64-w64-mingw32.exe",
+            "Tag Editor"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/Martchus/tageditor"
+    },
```
Comment (Ash258) [inline bucket/tageditor.json:20]:
```suggestion
    "checkver": "githbu",
```

File: bucket/tageditor.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/Martchus/tageditor/releases/download/v$version/tageditor-$version-x86_64-w64-mingw32.exe.zip"
+            }
```
Comment (Ash258) [inline bucket/tageditor.json:25]: 32bit version...

**Review by Ash258 — CHANGES_REQUESTED** [review]
Why there are no msvc builds??

File: bucket/tageditor.json [hunk 6]
```diff
@@ -7,11 +7,17 @@
         "64bit": {
             "url": "https://github.com/Martchus/tageditor/releases/download/v3.4.2/tageditor-3.4.2-x86_64-w64-mingw32.exe.zip",
             "hash": "ff4725dd63f5b8961dedd051c4b22f635d791f337adc2fc43c576e93c9250a5c"
+        },
+        "32bit": {
+            "url": "https://github.com/Martchus/tageditor/releases/download/v3.4.2/tageditor-3.4.2-i686-w64-mingw32.exe.zip",
+            "hash": "0163b9f1fd0cd44bb18887fbeecdd66de461113735ea704de23e54e0a4a98e6a"
         }
     },
+    "pre_install": "Get-ChildItem \"$dir\\tageditor*.exe\" | Rename-Item -NewName \"$dir\\tageditor.exe\"",
```
Comment (Ash258) [inline bucket/tageditor.json:16]: - Gci is overkill. Explicitly pass the executable name to -Path
- Rename item does not take full path

https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/rename-item?view=powershell-7.1#description

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/tageditor.json [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+        "64bit": {
+            "url": "https://github.com/Martchus/tageditor/releases/download/v3.4.2/tageditor-3.4.2-x86_64-w64-mingw32.exe.zip",
+            "hash": "ff4725dd63f5b8961dedd051c4b22f635d791f337adc2fc43c576e93c9250a5c"
+        },
+        "32bit": {
+            "url": "https://github.com/Martchus/tageditor/releases/download/v3.4.2/tageditor-3.4.2-i686-w64-mingw32.exe.zip",
+            "hash": "0163b9f1fd0cd44bb18887fbeecdd66de461113735ea704de23e54e0a4a98e6a"
+        }
+    },
+    "pre_install": "Rename-Item $dir\\tageditor*.exe $dir\\tageditor.exe",
```
Comment (Ash258) [inline bucket/tageditor.json:16]: - Why did you remove the quotes?
- Rename-Item does not take wildcard
- Rename-Item does not take fullpath for -NewName

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Comment by KaranKad** [comment]
Any reason why `"pre_install": "Get-ChildItem \"$dir\\tageditor*.exe\" | Rename-Item -NewName \"$dir\\tageditor.exe\"",` won't do. 
I tried to find closest application that matches this applications condition's and I found [Godot](https://github.com/lukesampson/scoop-extras/blob/master/bucket/godot.json)

- Portable.
- Single binary directly inside .zip file without additional directories.
- Binary named with version.

Everything matches except download location, so I decided to see its `pre_install` script and adapt it to this application and it works just like Godot's, and successfully installs. I don't know too much about PowerShell scripting or how scoop handles stuff so please show correct changes by correcting my PR.

**Review by Ash258 — APPROVED** [review]

**Pull request merged**